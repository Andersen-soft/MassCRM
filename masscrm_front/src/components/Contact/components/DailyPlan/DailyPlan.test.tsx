import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import * as ReactReduxHooks from 'react-redux';
import { DailyPlan } from '.';

jest.mock('react-redux', () => ({
  useSelector: jest.fn(),
  useDispatch: jest.fn()
}));

jest.mock('src/selectors');

jest.spyOn(ReactReduxHooks, 'useSelector').mockReturnValueOnce(30);

describe('DailyPlan', () => {
  it('DailyPlan should render correctly', () => {
    expect(shallowToJson(shallow(<DailyPlan />))).toMatchSnapshot();
  });
});
