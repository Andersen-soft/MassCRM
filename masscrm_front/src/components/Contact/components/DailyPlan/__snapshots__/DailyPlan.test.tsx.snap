// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DailyPlan DailyPlan should render correctly 1`] = `
<div
  className="daily-plan"
>
  <LabelIconGroup
    icon={
      Object {
        "$$typeof": Symbol(react.memo),
        "compare": null,
        "displayName": "PermIdentityIcon",
        "muiName": "SvgIcon",
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        },
      }
    }
    label="Daily plan"
  />
  <LabelIconGroup
    icon={
      Object {
        "$$typeof": Symbol(react.memo),
        "compare": null,
        "displayName": "PermIdentityIcon",
        "muiName": "SvgIcon",
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        },
      }
    }
    label="Complete"
  />
</div>
`;
